### PR TITLE
docs(sample): add missing samples (obj)

### PR DESCRIPTION
**Pull request by AlisskaPie**
Second part (2/3)
Samples to add with following region tags:
- [x] storage_bucket_delete_default_kms_key
- [x] storage_generate_encryption_key 
- [x] storage_view_bucket_iam_members
- [x] storage_add_bucket_label
- [ ] storage_compose_file
- [x] storage_remove_bucket_label
- [ ] storage_change_file_storage_class
- [x] storage_enable_versioning
- [ ] storage_set_metadata
- [x] storage_set_bucket_public_iam
- [x] storage_define_bucket_website_configuration
- [ ] storage_download_public_file
- [x] storage_get_service_account
- [x] storage_cors_configuration
- [x] storage_disable_versioning
- [x] storage_copy_file_archived_generation
- [ ] storage_object_csek_to_cmek
- [ ] storage_change_default_storage_class
- [x] storage_delete_file_archived_generation
- [x] storage_remove_cors_configuration

### CODE REVIEW

File: storage/objects/objects_test.go [hunk 1]
```diff
@@ -133,6 +134,13 @@ func TestObjects(t *testing.T) {
 		}
 	}
 
+	if err := copyFileArchivedGeneration(ioutil.Discard, bucketVersioning, bucketVersioning, object1, gen); err != nil {
+		t.Fatalf("copyFileArchivedGeneration: %v", err)
+	}
+	// delete first version of an object1 for a bucketVersioning.
```
Comment (IlyaFaer) [inline storage/objects/objects_test.go]: the first version

File: storage/objects/objects_test.go [hunk 2]
```diff
@@ -133,6 +134,13 @@ func TestObjects(t *testing.T) {
 		}
 	}
 
+	if err := copyFileArchivedGeneration(ioutil.Discard, bucketVersioning, bucketVersioning, object1, gen); err != nil {
```
Comment (IlyaFaer) [inline storage/objects/objects_test.go]: Two similar `bucketVersioning` args?

Comment (AlisskaPie) [reply storage/objects/objects_test.go]: Yes, to copy object inside the bucket. I saw in Java they did [this way](https://cloud.google.com/storage/docs/using-object-versioning#copy), but they didn't pass `dstBucket` in args. In c++, they copy from one bucket to another.
I'm not sure about keeping it in test, but it's comfortable 'cause we don't use `bucketVersioning` for other steps except disabling version and deleting, and other buckets we use.

File: storage/objects/objects_test.go (hunk 1 above)
Comment (AlisskaPie) [reply storage/objects/objects_test.go]: Done

File: storage/objects/objects_test.go (hunk 2 above)
Comment (IlyaFaer) [reply storage/objects/objects_test.go]: I mean, it looks like you're copying an object to the same bucket. Don't think it's very different from copy into another bucket, but who knows

**Pull request closed without merging**